### PR TITLE
Remove 'download' argument from Skript command

### PR DESCRIPTION
**Pull request by Asleeepp**
### Description

This PR aims to remove the redundant 'download' argument from '/skript update'
the 'download' argument did the exact same thing as the 'check' argument, so it didn't really make sense to have it, it's also misleading if it just checks instead of downloads the latest update.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/main/java/ch/njol/skript/SkriptCommand.java [hunk 1]
```diff
@@ -351,8 +351,6 @@ else if (args[0].equalsIgnoreCase("update")) {
 					updater.updateCheck(sender);
 				} else if (args[1].equalsIgnoreCase("changes")) {
 					updater.changesCheck(sender);
-				} else if (args[1].equalsIgnoreCase("download")) {
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/skript/SkriptCommand.java:354]: It needs removed from the CommandHelp object at the top of this file

**Review by UnderscoreTud — CHANGES_REQUESTED** [review]
Update the lang files to reflect this change

File: src/main/resources/lang/english.lang [hunk 2]
```diff
@@ -43,7 +43,6 @@ skript command:
 			description: Check for updates, read the changelog, or download the latest version of Skript
```
Comment (UnderscoreTud) [inline src/main/resources/lang/english.lang:43]: You missed the command description

**Review by UnderscoreTud — CHANGES_REQUESTED** [review]

File: src/main/resources/lang/english.lang (hunk 2 above)
Comment (Asleeepp) [reply src/main/resources/lang/english.lang:43]: 😭

Comment (Asleeepp) [reply src/main/resources/lang/english.lang:43]: i can't really do it for other languages, so i guess it'll just have to be english

File: src/main/resources/lang/german.lang [hunk 3]
```diff
@@ -43,7 +43,6 @@ skript command:
 			description: Überprüft auf eine neue Version, listet Änderungen dieser auf, und kann sie auch gleich herunterladen
```
Comment (Efnilite) [inline src/main/resources/lang/german.lang:43]:
```suggestion
			description: Überprüft auf eine neue Version oder listet Änderungen dieser auf
```

File: src/main/resources/lang/dutch.lang [hunk 4]
```diff
@@ -43,7 +43,6 @@ skript command:
 			description: Controleert of er updates zijn, laat de lijst met wijzigingen zien of downloadt de nieuwste versie van Skript
```
Comment (Efnilite) [inline src/main/resources/lang/dutch.lang:43]:
```suggestion
			description: Controleert of er updates zijn of laat de lijst met wijzigingen zien
```

**Review by Efnilite — APPROVED** [review]

**Review by cheeezburga — APPROVED** [review]

File: src/main/resources/lang/french.lang [hunk 5]
```diff
@@ -43,7 +43,6 @@ skript command:
 			description: Vérifie les mises à jour, liste le journal des modifications ou télécharge la dernière version de Skript
```
Comment (sovdeeth) [inline src/main/resources/lang/french.lang:43]:
```suggestion
			description: Vérifie les mises à jour ou liste le journal des modifications.
```

**Review by sovdeeth — APPROVED** [review]

**Review by APickledWalrus — APPROVED** [review]
Seems fine

**Pull request merged**